build-output-api/isr-revalidation/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,37 @@
+const https = require('https');
+
+// The bypass token can be a randomly generated string of at least 32 characters.
+// This is meant to be *private* - DO NOT expose this value on the client-side.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+module.exports = (req, res) => {
```

Comment:
If this function were an edge function could it use `fetch`?

Reply:
Yes! You could require a fetch library and use it here. I was doing that in an earlier version of this PR: https://github.com/vercel/examples/blob/87e3f730493d94cce300594cdd52b9c6083d66ea/build-output-api/isr-revalidation/.vercel/output/functions/revalidate.func/index.js

Reply:
for edge functions we still need to polyfill `fetch`? 😮 My hope was no as that would make this function simpler I guess.

Reply:
Oh, I misread. If this were an Edge Function, we could use `fetch` directly, yes. Sorry for the confusion!